_includes/checklist.md

```diff
@@ -25,7 +25,7 @@ Everything in this section needs to be completed before the project will be sche
         * Search [this page](https://insite.gsa.gov/portal/content/627238) for "Rules of Engagement (RoE) 90-Day LATO Penetration Test TEMPLATE", even if this isn't for a 90-day LATO.
     - [ ] Add [System Security Plan (SSP)](https://before-you-ship.18f.gov/ato/ssp/) template
         * For Low systems on cloud.gov, use [this template](https://docs.google.com/a/gsa.gov/document/d/1tVbH39TFfvSaBbjWfLaR3GLOuvsLuhLFJ75xKowEV5c/edit?usp=sharing)
-        * Not applicable to 90-day ATOs.
+        * For a 90-day ATO, delete Section 13.
```

Comment:
@hollyallen Is this correct? This is contrary to my previous understanding, but just came up in the sprint working session.